global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "7.0.101"
+        "version": "7.0.103"
```

Comment:
We should update this and https://github.com/microsoft/fhir-server/blob/main/build/docker/Dockerfile#L1, maybe in a different PR. Is there a good way to automate this?

Reply:
Thanks for the comment.